prisma/schema.prisma

```diff
@@ -1,39 +1,37 @@
-
 generator client {
   provider = "prisma-client-js"
 }
 
 datasource db {
-  provider = "mongodb"
-  url      = env("DATABASE_URL")
+  provider     = "mongodb"
+  url          = env("DATABASE_URL")
   relationMode = "prisma"
-
 }
 
 model User {
-  id             String    @id @default(auto()) @map("_id") @db.ObjectId
-  name           String?
-  handle         String?   @unique
-  bio            String?
-  image          String?  
-  email          String?   @unique
-  emailVerified  DateTime?
-  totalViews     Int       @default(0)
-  createdAt      DateTime  @default(now())
-  updatedAt      DateTime  @updatedAt
-  links          Link[]
+  id            String    @id @default(auto()) @map("_id") @db.ObjectId
+  name          String?
+  handle        String?   @unique
+  bio           String?
+  image         String?
+  banner        String?
```

Comment:
Adding a banner to the user's page goes against my initial plan of making the site as simple as possible

Reply:
Yeah! I agree.